src/api/config/environments/development.rb

```diff
@@ -11,6 +11,13 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Eager load BsRequest and HistoryElement variants, otherwise Rails do not load the STI subclasses in development.
+  sti_classes_to_eager_load = Dir["app/models/bs_request/**/*.rb", "app/models/history_element/**.rb"]
```

Comment:
Hm there are more STI classes right?

Reply:
Yes, but I approved the PR since this is only for handling missing history elements. I guess a way to list all STI classes without having to update this would be even better, but I don't think it's possible.

Reply:
So let's do what is possible I guess? 😃

Reply:
I've added all the STI classes to the eager load array

Reply:
but yeah this is probably going to far for this PR...